challengeMk2/ViewModels/StarSystemsViewModel.cs

```diff
@@ -80,20 +82,22 @@ void InitializeDatabase()
 
         async Task<List<StarSystem>> GetAndSaveDataFromApi()

```

Comment:
When you have an `And` in the name of a method, it's a sign it's doing too much. Try splitting the method in several smaller ones.

Reply:
i split the big method in 2, made new method to SaveDataAsync() only. But this new method still called by the first one.
Should i take out the second method from the first (but if do so, have to create a variable to store retrieved data before save them )?